frontend/src/components/GeocacheList/GeocacheList.tsx

```diff
@@ -1,7 +1,93 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { Geocache } from "../../model/Geocache";
+
 const GeocacheList = () => {
+    const [notes, setNotes] = useState([]);
```

Comment:
Maybe the variable name would be more descriptive if it was geocaches instead of notes